rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -1262,6 +1262,10 @@ stream {
             {{ buildModSecurityForLocation $all.Cfg $location }}
 
             {{ if isLocationAllowed $location }}
+            {{ if gt (len $location.Denylist.CIDR) 0 }}
+            {{ range $ip := $location.Denylist.CIDR }}
+            deny {{ $ip }};{{ end }}
+            {{ end }}
```

Comment:
I should move this part to Whitelist, refer to:
https://github.com/kubernetes/ingress-nginx/pull/7121/files#diff-cbf382cc05c9f274b5db56a581b335dba8ecb80fd96a8f1a6a068b2594c9b1caR1171-R1174